core/broadleaf-framework/src/main/resources/bl-framework-applicationContext.xml

```diff
@@ -257,4 +257,39 @@
         <property name="collectionRef" value="blMergedCacheConfigLocations-framework"/>
         <property name="targetRef" value="blMergedCacheConfigLocations"/>
     </bean>
+
```

Comment:
I think you need to remove these. There is error `Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'blProductInventoryUseDefaultSku' available`.